json_test.go

```diff
@@ -10,7 +10,34 @@ import (
 	"github.com/rocktavious/autopilot/v2023"
 )
 
-var validStringContainingJSON = `{"name":"Thomas","isIntern":false,"age":45,"access":{"aws":"admin","okta":"admin"},"tags":["org:engineering","team:platform"]}`
```

Comment:
Remove the `"age":45,` here because it is annoying to have `reflect.DeepEqual` fail to compare a int to a float64. This is handled by JSON marshal, we don't get any benefits from testing this on our own. We just need to make sure that keys and values are not being discarded/changed.